.github/workflows/validate_samples.yml

```diff
@@ -0,0 +1,25 @@
+name: Validate-Samples
+
+on:
+  push:
+    paths:
+      - 'data/*/TIDES/*'
+      - 'spec/*'
+  pull_request:
+    paths:
+      - 'data/*/TIDES/*'
+      - 'spec/*'
+  workflow_dispatch:
+  create:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Validate data
+        uses: frictionlessdata/repository@v2
```

Comment:
Is there any way to pass `--schema-sync` to this action? Otherwise it will fail when option fields are left out of the csvs.

Reply:
looking into it